bitrise.yml

```diff
@@ -53,6 +53,10 @@ workflows:
             else
               echo "VDTESTING_DOWNLOADED_FILES_DIR: $VDTESTING_DOWNLOADED_FILES_DIR"
             fi
+    - deploy-to-bitrise-io@2:
+        inputs:
+        - is_compress: "true"
+        - deploy_path: $VDTESTING_DOWNLOADED_FILES_DIR
```

Comment:
Did you add this to try out something while you were developing or is not now required for the test workflow to run?